notebooks/MAST/plotting_quasar_observations/plotting_quasar_observations.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<p>I think based off of one of the recent examples given (https://github.com/ttdu/notebooks/blob/JWST-EDB-Query/notebooks/MAST/JWST/Engineering_Database_Retreival/EDB_Retrieval.ipynb) we don't need to use the "Main Content" header, but we can start using however many 1st-level headers we want (with additional subheaders). So "Gather the Data" can be the higher-level header. Maybe "Query the MAST Archive by Object Name" is more descriptive?</p><p> </p><p>The tutorial template says "Try to weave resource links into the main content of your tutorial so that they are falling in line with the context of your writing." So you can add that link here; particularly if you want to tell the user any particular part of it to look at to be helpful right now.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>